ncm-spma/pom.xml

```diff
@@ -27,6 +27,8 @@
   <properties>
     <spma.ips.author>Mark R. Bannister</spma.ips.author>
     <spma.ips.email>Mark.Bannister@morganstanley.com</spma.ips.email>
+    <spma.yum-ng.author>Jindrich Novy</spma.yum-ng.author>
```

Comment:
In fact is is 'yumng' without the dash - as the suffix is added from /software/components/spma/suffix which is 'ng' -> yumng.